templates/components/multiagent/typescript/workflow/stream.ts

```diff
@@ -0,0 +1,57 @@
+import { WorkflowEvent } from "@llamaindex/core/workflow";
+import {
+  createCallbacksTransformer,
+  createStreamDataTransformer,
+  StreamData,
+  trimStartOfStreamHelper,
+  type AIStreamCallbacksAndOptions,
+} from "ai";
+import { AgentRunResult, FunctionCallingStreamResult } from "./type";
+
+export function toDataStream(
```

Comment:
Can we use the LlamaIndexAdapter like this?
1. If a workflow returns a `AsyncGenerator<WorkflowEvent, void>` this implements an `AsyncIterable<WorkflowEvent>`
2. If we create a `WorkflowEvent` with `delta: string` as property that event implements `EngineResponse` (so it could be consumed with the `LlamaIndexAdapter`)